src/hockeypuck/conflux/recon/settings.go

```diff
@@ -331,27 +335,30 @@ func (c *PTreeConfig) NumSamples() int {
 
 // RandomPartnerAddr returns the a weighted-random chosen resolved network
 // addresses of configured partner peers.
-func (s *Settings) RandomPartnerAddr() (net.Addr, error) {
+func (s *Settings) RandomPartnerAddr() (net.Addr, []error) {
```

Comment:
Consider `go.uber.org/multierr` for this kind of thing, I think it'll make things a little easier.

Reply:
I'll look into that.